src/utils/Utils.ts

```diff
@@ -872,6 +872,10 @@ export default class Utils {
     return !Object.keys(obj).length;
   }
 
+  public static isEmptyString(str: string): boolean {
+    return str ? str.length === 0 : true;
```

Comment:
[low] If this returns true, it means 'null' string and not 'empty' string ;-)
So the method name should be called isNullOrEmptyString()
I'll change it.

Reply:
btw an empty string is considered as null in JS :-)
So 'str.length === 0' will never be evaluated if 'str' equals to ""